src/test/scripts/helm_install.sh

```diff
@@ -123,17 +127,33 @@ bootstrap_database() {
      --set postgresqlPassword="$PRODUCT_NAME" \
      --version "$POSTGRES_CHART_VERSION" \
      $HELM_DEBUG_OPTION \
-     bitnami/postgresql > $LOG_DOWNLOAD_DIR/helm_install_log.txt
+     bitnami/postgresql >> $LOG_DOWNLOAD_DIR/helm_install_log.txt
 
   if [[ "$DB_INIT_SCRIPT_FILE" ]]; then
     kubectl cp -n "${TARGET_NAMESPACE}" $DB_INIT_SCRIPT_FILE $POSTGRES_RELEASE_NAME-0:/tmp/db-init-script.sql
     kubectl exec -n "${TARGET_NAMESPACE}" ${POSTGRES_RELEASE_NAME}-0 -- /bin/bash -c "psql postgresql://$PRODUCT_NAME:$PRODUCT_NAME@localhost:5432/$DB_NAME -f /tmp/db-init-script.sql"
   fi
 }
 
+bootstrap_elasticsearch() {
```

Comment:
Out of interest, are we using basic auth for ES? If so where are the credentials defined?

Reply:
We don't use auth or HTTPS (yet). The open-source version of ES doesn't support it, and the ways of hacking it in are not ideal. I wrote up some notes here: https://hello.atlassian.net/wiki/spaces/DCD/pages/1252068866/Notes+Bitbucket+Elasticsearch+recommendations

Medium term it is _very_ likely we will support OpenSearch for external ES (as AWS will be moving to it), so we can migrate to that then.